src/components/react-admin/MapPointInput.js

```diff
@@ -5,12 +5,12 @@ import { addField } from 'react-admin';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { withStyles } from '@material-ui/core/styles';
 
-import compose from '../../utils/compose';
+import { getSettings } from '../../services/settings';
+import { connectAppProvider } from '../AppProvider';
 
+import compose from '../../utils/compose';
 
-const Map = ReactMapboxGl({
-  accessToken: 'pk.eyJ1IjoibWFraW5hY29ycHVzIiwiYSI6ImNqY3E4ZTNwcTFta3ozMm80d2xzY29wM2MifQ.Nwl_FHrWAIQ46s_lY0KNiQ',
-});
+let Map = null; // For the mapbox gl component
```

Comment:
As you load it from `useEffect` hook, maybe storing it into component state would be preferable, wouldn't it?

Reply:
*(Also loaded state wouldn't be required anymore as you could simply check if map (state) is truthy or not.)*

Reply:
I tried, but it failed inside ReactMapboxGL code. I think the map object is not suitable for use as a component state, but why ?

Reply:
We can see this later if you want.